sdk/textanalytics/Azure.AI.TextAnalytics/tests/RetryOnInternalServerErrorAttributeTests.cs

```diff
@@ -14,18 +14,24 @@ namespace Azure.AI.TextAnalytics.Tests
     public class RetryOnInternalServerErrorAttributeTests
     {
         private const string FailedToProcessTaskExceptionMessage =
-            "Failed to process task after several retry,"
-            + " Status: 200 (OK),"
-            + " ErrorCode: InternalServerError";
+            "Failed to process task after several retry"
```

Comment:
"With great power comes great responsibility." :)

Have you been letting the service team know about these issues? When I initially did something like this with Key Vault, it was after effectively triaging the sporadic issues we were finding and investigations on their side.

Reply:
Yes, I filed bugs for each of these and provided all the information that I have.